content/en/docs/Writing policies/jmespath.md

```diff
@@ -434,51 +442,1271 @@ spec:
+}
+```
+
+```json
+{
+  "color": "black",
+  "dog": "lab"  
+}
+```
+
+| Input 1            | Input 2            | Output   |
+|--------------------|--------------------|----------|
+| Map (Object)       | Map (Object)       | Boolean  |
+
+<br>
+
+**Example:** This policy checks all incoming Deployments to ensure they have a matching, preexisting PodDisruptionBudget in the same Namespace. The `label_match()` filter is used in a query to count how many PDBs have a label set matching that of the incoming Deployment.
+
+```yaml
+apiVersion: kyverno.io/v1
+kind: ClusterPolicy
+metadata:
+  name: require-pdb
+spec:
+  validationFailureAction: audit
+  background: false
+  rules:
+  - name: require-pdb
+    match:
+      any:
+      - resources:
+          kinds:
+          - Deployment
+    preconditions:
+      any:
+      - key: "{{request.operation}}"
+        operator: Equals
+        value: CREATE
+    context:
+    - name: pdb_count
+      apiCall:
+        urlPath: "/apis/policy/v1beta1/namespaces/{{request.namespace}}/poddisruptionbudgets"
+        jmesPath: "items[?label_match(spec.selector.matchLabels, `{{request.object.spec.template.metadata.labels}}`)] | length(@)"
+    validate:
+      message: "There is no corresponding PodDisruptionBudget found for this Deployment."
+      deny:
+        conditions:
+          any:
+          - key: "{{pdb_count}}"
+            operator: LessThan
+            value: 1
+```
+
+</p>
+</details>
+
 ### Modulo
 
-### Multiply
+<details><summary>Expand</summary>
+<p>
 
-### Parse_json
+The `modulo()` filter returns the [modulo](https://www.khanacademy.org/computing/computer-science/cryptography/modarithmetic/a/what-is-modular-arithmetic) or remainder between a division of two numbers. For example, the modulo of a division between `10` and `3` would be `1` since `3` can be divided into `10` only `3` times (equaling `9`) while producing `1` as a remainder.
 
-### Parse_yaml
+Arithmetic filters like `modulo()` currently accept inputs in the following formats.
 
-### Path_canonicalize
+* Number (ex., \`10\`)
+* Quantity (ex., '10Mi')
+* Duration (ex., '10h')
 
-### Pattern_match
+Note that how the inputs are enclosed determines how Kyverno interprets their type. Numbers enclosed in back ticks are scalar values while quantities and durations are enclosed in single quotes thus treating them as strings. Using the correct enclosing character is important because, in Kubernetes "regular" numbers are treated implicitly as units of measure. The number written \`10\` is interpreted as an integer or "the number ten" whereas '10' is interpreted as a string or "ten bytes". See the [Formatting](#formatting) section above for more details.
 
-### Regex_match
+| Input 1            | Input 2            | Output             |
+|--------------------|--------------------|--------------------|
+| Number             | Number             | Number             |
+| Quantity           | Quantity           | Quantity           |
```

Comment:
I would say that this isn't entirely a valid input set. Could we restrict the documentation to numbers for now. I would prefer to do more research on this before we document the behavior for quantities.

Reply:
Done. Will leave the note in place until we reach full resolution.